index.html

```diff
@@ -4,45 +4,240 @@
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&family=Roboto:wght@400;500;700&display=swap"
-      rel="stylesheet"
-    />
-    <link rel="stylesheet" href="./index.css" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Letaru Linda Masia</title>
   </head>
   <body>
-    <header class="header">
-      <nav class="top-nav">
-        <a href="#" class="logo">John Doe</a>
-        <img src="./Assets/Menu.svg" alt="humburger menu" />
+    <header class="site-header">
+      <a href="#"> Letaru Linda Masia</a>
+      <nav class="nav-menu">
+        <img src="./images/Vector.png" alt="Vector image " />
       </nav>
     </header>
-    <section class="background-one">
-      <h1 class="introduction">Hey There.<br />I'm Linda</h1>
-      <h3 class="profession">I'am a software Developer</h3>
-      <p class="content">
-        I can help you build a product,Feature or website.Lookthrough some of my
-        work and experience !If you like what you see and have a project you
-        need coded,<br />dont hesistate to conduct me.
-      </p>
-      <div class="social">
-        <ul class="social-links">
-          <li>
-            <img src="./Assets/github.svg" alt="Github symbol" />
-          </li>
-          <li><img src="./Assets/linkedin.svg" alt="Linkedin symbol" /></li>
-          <li><img src="./Assets/angelist.svg" alt="Angelic Symbol" /></li>
-          <li>
-            <img src="./Assets/Twitter.svg" alt="Twitter sysmbol" />
-          </li>
-          <li>
-            <img src="./Assets/medium.svg" alt="Medium symbol" />
-          </li>
-        </ul>
-      </div>
-    </section>
+    <main>
+      <section class="container">
+        <div class="text">
+          <h1>Hey There. I'm Linda</h1>
+          <h3>I'm a Software Developer.</h3>
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+        </div>
+        <div class="icon">
+          <ul>
+            <li>
+              <a href="#"><img src="./images/img1.png" alt="Github-icon" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/img2.png" alt="LinkeIn-icon" /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="./images/img3.png" alt="AngelList-icon"
+              /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/img4.png" alt="Twitter-icon" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/img5.png" alt="Medium-icon" /></a>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section class="work-container">
```

Comment:
- [ ] Would you please try as much as possible to follow the figma design. Your `work section` does not have the   images and styling to match the Figma design. Keep in mind that this is a project requirement.
### your design

![image](https://user-images.githubusercontent.com/98453979/185763474-2d0087e4-f6fa-469e-839d-27bddbaa46c2.png)

### Figma design

![image](https://user-images.githubusercontent.com/98453979/185763544-b013dfcd-6e49-4159-9174-ad43071448b7.png)